src/main/java/com/example/onlinebookstore/validation/StatusValidator.java

```diff
@@ -0,0 +1,17 @@
+package com.example.onlinebookstore.validation;
+
+import com.example.onlinebookstore.model.Order;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class StatusValidator implements ConstraintValidator<ValidStatus, Order.Status> {
+    @Override
+    public boolean isValid(Order.Status status, ConstraintValidatorContext context) {
+        try {
+            Order.Status.valueOf(String.valueOf(status));
```

Comment:
what the purpose of this validator?

Reply:
check if the status of the request is correct